### PR TITLE
ENG-969 Configure code evaluation tooling

### PR DESCRIPTION
**Pull request by uberflip-aviel**
Research/get tools pre-installed in gitpod environment for evaluation:

* review eslint rules in place
* cyclomatic complexity checks?
* maintainability index?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "require": {
+        "squizlabs/php_codesniffer": "*"
+    },
+    "scripts": {
+        "phpcs": "./vendor/bin/phpcs --config-set default_standard PSR2"
```
Comment (uberflip-jbooth) [inline composer.json]: Personally I dislike commands like `./vendor/bin/blahblahblah`.  I would put additional scripts in composer.json to wrap phpcs.  e.g. change the one you have to `phpcsinit` and add `phpcs` and `phpcbf` so all I need to do is type `composer phpcs /path/of/file`

File: README.md [hunk 2]
```diff
@@ -67,3 +67,10 @@ Available on standard ports (eg. `5672` or `5552`) via `localhost`. The manageme
 | vhost     | /         |
 | username  | guest     |
 | password  | guest     |
+
+### PHP CodeSniffer (Linter)
+
+1. Run `./vendor/bin/phpcs /path/of/file` to evaluate the code
+2. Run `./vendor/bin/phpcbf /path/of/file` to fix code using default coding standard
+
+Note: Default coding standard is set to PSR2
```
Comment (uberflip-jbooth) [inline README.md:76]: Have you run the linter on a sample file to make sure it works as expected?

Comment (uberflip-aviel) [reply README.md:76]: Yes I have. I did create a small php code. It returns all the errors and warnings, as well as fixing all those errors using phpcbf.

File: composer.json (hunk 1 above)
Comment (uberflip-aviel) [reply composer.json]: Good point! I'll try to make some changes

**Pull request merged**